[Bug]:  Could not pass event PlayerTeleportEvent to Multiverse-Core v4.3.1-b861

### `/mv version -p` output

Multiverse-Core v4.3.1-b861

### Server logs

org.bukkit.event.EventException: null         at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[arclight-1.20.1-1.0.1-SNAPSHOT-041b0b1.jar%23238!/:arclight-1.20.1-1.0.1-SNAPSHOT-041b0b1]         at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:589) ~[arclight-1.20.1-1.0.1-SNAPSHOT-041b0b1.jar%23238!/:arclight-1.20.1-1.0.1-SNAPSHOT-041b0b1]         at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:576) ~[arclight-1.20.1-1.0.1-SNAPSHOT-041b0b1.jar%23238!/:arclight-1.20.1-1.0.1-SNAPSHOT-041b0b1]         at org.bukkit.craftbukkit.v1_20_R1.entity.CraftPlayer.teleport(CraftPlayerMixin.java:921) ~[arclight-1.20.1-1.0.1-SNAPSHOT-041b0b1.jar%23238!/:arclight-1.20.1-1.0.1-SNAPSHOT-041b0b1]         at com.Zrips.CMI.Modules.tp.Teleportations.finalizeTeleport(Teleportations.java:979) ~[?:?]         at com.Zrips.CMI.Modules.tp.Teleportations.teleport(Teleportations.java:450) ~[?:?]         at com.Zrips.CMI.commands.list.back.perform(back.java:74) ~[?:?]         at com.Zrips.CMI.commands.CommandsHandler.onCommand(CommandsHandler.java:398) ~[?:?]         at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45) ~[arclight-1.20.1-1.0.1-SNAPSHOT-041b0b1.jar%23238!/:arclight-1.20.1-1.0.1-SNAPSHOT-041b0b1]         at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:149) ~[arclight-1.20.1-1.0.1-SNAPSHOT-041b0b1.jar%23238!/:arclight-1.20.1-1.0.1-SNAPSHOT-041b0b1]         at org.bukkit.craftbukkit.v1_20_R1.CraftServer.dispatchCommand(CraftServerMixin.java:875) ~[arclight-1.20.1-1.0.1-SNAPSHOT-041b0b1.jar%23238!/:arclight-1.20.1-1.0.1-SNAPSHOT-041b0b1]         at org.bukkit.craftbukkit.v1_20_R1.command.BukkitCommandWrapper.run(BukkitCommandWrapper.java:50) ~[arclight-1.20.1-1.0.1-SNAPSHOT-041b0b1.jar%23238!/:arclight-1.20.1-1.0.1-SNAPSHOT-041b0b1]         at com.mojang.brigadier.CommandDispatcher.execute(CommandDispatcher.java:264) ~[brigadier-1.1.8.jar%23110!/:?]         at net.minecraft.commands.Commands.m_242674_(CommandsMixin.java:257) ~[server-1.20.1-20230612.114412-srg.jar%23315!/:?]         at net.minecraft.server.network.ServerGamePacketListenerImpl.m_246958_(ServerPlayNetHandlerMixin.java:2767) ~[server-1.20.1-20230612.114412-srg.jar%23315!/:?]         at net.minecraft.server.network.ServerGamePacketListenerImpl.m_244885_(ServerPlayNetHandlerMixin.java:1221) ~[server-1.20.1-20230612.114412-srg.jar%23315!/:?]         at net.minecraft.util.thread.BlockableEventLoop.m_18693_(BlockableEventLoop.java:67) ~[server-1.20.1-20230612.114412-srg.jar%23315!/:?]         at java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768) ~[?:?]         at net.minecraft.server.TickTask.run(TickTask.java:18) ~[server-1.20.1-20230612.114412-srg.jar%23315!/:?]         at net.minecraft.util.thread.BlockableEventLoop.m_6367_(BlockableEventLoop.java:156) ~[server-1.20.1-20230612.114412-srg.jar%23315!/:?]         at net.minecraft.util.thread.ReentrantBlockableEventLoop.m_6367_(ReentrantBlockableEventLoop.java:23) ~[server-1.20.1-20230612.114412-srg.jar%23315!/:?]         at net.minecraft.server.MinecraftServer.m_6367_(MinecraftServerMixin.java:770) ~[server-1.20.1-20230612.114412-srg.jar%23315!/:?]         at net.minecraft.server.MinecraftServer.m_6367_(MinecraftServerMixin.java:161) ~[server-1.20.1-20230612.114412-srg.jar%23315!/:?]         at net.minecraft.util.thread.BlockableEventLoop.m_7245_(BlockableEventLoop.java:130) ~[server-1.20.1-20230612.114412-srg.jar%23315!/:?]         at net.minecraft.server.MinecraftServer.m_129961_(MinecraftServerMixin.java:753) ~[server-1.20.1-20230612.114412-srg.jar%23315!/:?]         at net.minecraft.server.MinecraftServer.m_7245_(MinecraftServerMixin.java:747) ~[server-1.20.1-20230612.114412-srg.jar%23315!/:?]         at net.minecraft.util.thread.BlockableEventLoop.m_18699_(BlockableEventLoop.java:115) ~[server-1.20.1-20230612.114412-srg.jar%23315!/:?]         at net.minecraft.server.MinecraftServer.m_130012_(MinecraftServerMixin.java:732) ~[server-1.20.1-20230612.114412-srg.jar%23315!/:?]         at net.minecraft.server.MinecraftServer.m_130011_(MinecraftServerMixin.java:2074) ~[server-1.20.1-20230612.114412-srg.jar%23315!/:?]         at net.minecraft.server.MinecraftServer.m_206580_(MinecraftServerMixin.java:251) ~[server-1.20.1-20230612.114412-srg.jar%23315!/:?]         at java.lang.Thread.run(Thread.java:1589) ~[?:?] Caused by: java.lang.NullPointerException: Cannot invoke "org.bukkit.command.CommandSender.getName()" because "teleporter" is null         at com.onarandombox.MultiverseCore.listeners.MVPlayerListener.playerTeleport(MVPlayerListener.java:233) 

### Server Version

```shell
1.20.1 Aarclight
```


### Bug Description

一直报错

### Steps to reproduce

当玩家进入豆腐世界的时候，就弹出这个

### Agreements

- [X] I have searched for and ensured there isn't already an open or resolved issue(s) regarding this.
- [X] I was able to reproduce my issue on a freshly setup and up-to-date server with the latest version of Multiverse plugins with no other plugins and with no kinds of other server or client mods.

--- a/src/main/java/com/onarandombox/MultiverseCore/listeners/MVPlayerListener.java
+++ b/src/main/java/com/onarandombox/MultiverseCore/listeners/MVPlayerListener.java
@@ -170,7 +170,7 @@ public class MVPlayerListener implements Listener {
             return;
         }
         Player teleportee = event.getPlayer();
-        CommandSender teleporter = null;
+        CommandSender teleporter = teleportee;
         String teleporterName = MultiverseCore.getPlayerTeleporter(teleportee.getName());
         if (teleporterName != null) {
             if (teleporterName.equals("CONSOLE")) {

